### ISSUE TITLE
incorrect view and export of polycyclic aromatic compounds

### ISSUE THREAD

**Issue by rmrmg** [partridgejiang/Kekule.js#30]
I selected benzene ring from menu and after two clicks I end up with structure presented in the picture below.
![naphtalene](https://user-images.githubusercontent.com/20069573/31892320-0903ec48-b809-11e7-8b2d-7b135a81002e.png). Also SMILES obtained for this is incorrect ( C1=CC=2=C(C=C1)=CC=CC=2 ).
Such a way (adding another benzene ring) is the most procedure to draw polycyclic aromatic hydrocarbons.

**Comment by partridgejiang** [comment]
Hi @rmrmg, it is a known issue in using the benzene button. Currently the editor simply add a benzene ring (always 3 single and 3 double bonds) regardless of its neighbor environment. To workaround it, you can use the cyclohexane button, then add essential double bond on it.

**Comment by rmrmg** [comment]
@partridgejiang: Yes, and other workaround is to remove unneeded double bonds. But both solutions are not very convenient for users, so I rather write a function to check and repair smiles obtained from the editor. Similar problem is with heteroatoms if you type any lower case symbol (o, n, ...) the editor places such symbol in smiles without any checking.  When you fix it, please let me know.

**Comment by partridgejiang** [comment]
The lowcase symbol is now regarded as a pseudo atom in editor and in outputting the SMILES, it will be replaced with '[*]' (e.g. C1=C[*]C=C1 if input 'n' instead of 'N' in pyrrole). Could you send me a few test cases that cause the lowcase symbol to be outputed for debugging? Thanks.

**Comment by rmrmg** [comment]
@partridgejiang:  You right, everywhere there are [*] not lower case letter. Sorry for my mistake.
